squad/ci/backend/lava.py

```diff
@@ -314,6 +319,9 @@ def __parse_results__(self, data, test_job):
                         res_value = result['measurement']
                         metrics.update({res_name: float(res_value)})
                 else:
+                    if ignore_lava_suite:
```

Comment:
IMHO this is wrong. The idea in #174 is to treat 'lava' as any other suite. Meaning that block 'if' above should be applied to all suites. So the proper solution should be also to apply 'ignore_lava_suite' to the if block above:
```
if ignore_lava_suite or result['suite'] != 'lava'
```

In this context variable name is misleading.

Reply:
Oh I missed that point! It definitely makes more sense. I'll rename it to something like 'handle_lava_suite'.